std/random.d

```diff
@@ -806,12 +941,17 @@ alias Mt19937 = MersenneTwisterEngine!(uint, 32, 624, 397, 31,
 @safe pure nothrow unittest //11690
 {
     alias MT(UIntType, uint w) = MersenneTwisterEngine!(UIntType, w, 624, 397, 31,
-                                                        0x9908b0df, 11, 7,
-                                                        0x9d2c5680, 15,
-                                                        0xefc60000, 18);
+                                                        0x9908b0df,
+                                                        11, 0xffffffff,
+                                                        7, 0x9d2c5680,
+                                                        15, 0xefc60000,
+                                                        18);
 
     foreach (R; std.meta.AliasSeq!(MT!(uint, 32), MT!(ulong, 32), MT!(ulong, 48), MT!(ulong, 64)))
+    {
         auto a = R();
+        a.seed(a.defaultSeed); // checks that some alternative paths in `seed` are utilized
```

Comment:
A note on this change: ideally it would be nice to include checks on `a.front` (before & after `popFrontN(9999)`).  I don't have any values from a reference implementation to compare to right now, but I'll try to follow up on this soon-ish.